scrapi/migrations.py

```diff
@@ -0,0 +1,36 @@
+import logging
+# import functools
+
+from scrapi import tasks
+from scrapi import settings
+from scrapi.linter import RawDocument
+from scrapi.processing.elasticsearch import es
+from celery import Celery
+
+app = Celery()
+app.config_from_object(settings)
+
+logger = logging.getLogger()
+
+
+@tasks.task_autoretry(default_retry_delay=30, max_retries=5)
+def rename(doc, source=None, target=None, dry=True):
```

Comment:
you may want to allow ```**kwargs``` here as well, just so that if you accidentally specified an extra option at the command line it wouldn't fail. Though we may want it to fail, just to make you be explicit.